source/NVDAObjects/window/excel.py

```diff
@@ -787,7 +880,12 @@ class ExcelCellTextInfo(NVDAObjectTextInfo):
 
 	def _getFormatFieldAndOffsets(self,offset,formatConfig,calculateOffsets=True):
 		formatField=textInfos.FormatField()
-		fontObj=self.obj.excelCellObject.font
+		if (self.obj.excelCellObject.Application.Version > "12.0"):
+					cellObj=self.obj.excelCellObject.DisplayFormat
+					fontObj=cellObj.font
+		else:
+            		cellObj=self.obj.excelCellObject
+            		fontObj=cellObj.font

```

Comment:
Indentation is a bit broken here. One line has 5 tabs where it should probably have 3 and some other lines have mixed spaces and tabs instead of just tabs.